### DISCUSSION TITLE
Unclear alt text on bot-posted images?

### DISCUSSION THREAD

**Discussion by JoshuaKGoldberg** [renovatebot/renovate#25714] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

~I am trying to get this working for the first time~ It's always been this way.

### Describe the problem

Right now, comments from the Renovate GitHub app start with Markdown like:

> ```md
> [![Mend Renovate](https://app.renovatebot.com/images/banner.svg)](https://renovatebot.com)
>
> This PR contains the following updates:
> ```

That results in HTML roughly like:

> ```html
> <img alt="Mend Renovate" data-canonical-src="..." src="..." style="max-width: 100%;">
> ```

Example PR: https://github.com/JoshuaKGoldberg/refined-saved-replies/pull/162

I'm not sure that the alt text of _"Mend Renovate"_ is clear enough to explain the image's intent to a user who can't visually perceive the image. The alt text is at the very least sparse enough to trigger https://github.com/github/accessibility-alt-text-bot's default settings:

> Uh oh! @renovate[bot], the image you shared is missing helpful alt text. Check your pull request body.
>
> Alt text is an invisible description that helps screen readers describe images to blind or low-vision users. If you are using markdown to display images, add your alt text inside the brackets of the markdown image.
>
> Learn more about alt text at [Basic writing and formatting syntax: images on GitHub Docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#images).

Proposal: can we modify the alt text on that image to something just a bit more informative? Say:

```diff
- > [![Mend Renovate](https://app.renovatebot.com/images/banner.svg)](https://renovatebot.com)
+ > [![Mend Renovate logo banner](https://app.renovatebot.com/images/banner.svg)](https://renovatebot.com)
```

Note that I'm not a user who typically uses a screenreader or other assistive technology. If all this discussion is wontfixed with the conclusion that the alt text bot is giving a false flag on the image, I'd be happy with that conclusion too. 🙂 

### Relevant debug logs

n/a

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment]
@Gabriel-Ladzaretti can we make this change in the app?

**Reply by Gabriel-Ladzaretti** [reply, marked as answer]
done

**Reply by JoshuaKGoldberg** [reply]
Amazing, thanks for the fast turnaround!
